main.js

```diff
@@ -67,9 +82,11 @@ function getSqlExecutor(httpRequestFieldName) {
     let rows = [];
     try {
       if (sql.toLowerCase().includes("select")) {
-        rows = db.prepare(sql).all();
+        var stmt = db.prepare(sql);
+        rows = stmt.all(params);
       } else {
-        db.prepare(sql).run();
+        var stmt = db.prepare(sql);
+        stmt.run(params);
```

Comment:
Could you use `const` and `let` instead of `var` throughout the code?

Reply:
for the stmt variable absolutely (bad habits are hard to break!). params must not be a const though.